NEWS.md

```diff
@@ -122,6 +122,14 @@ be directly added to this file to describe the related changes.
 - Added a new function for generating C++ header files for new module classes:
   `module_write`
 
+- The conversion of CO2 assimilation to biomass is no longer hard coded into
+  the photosynthesis modules: c3CanAC, CanAC, multilayer_canopy_integrator, etc.
+  These modules now produce canopy assimilation rates in micromol CO2 / m^2 / s
+  A new module called `biocro:carbon_assimilation_to_biomass` now performs the
```

Comment:
The module is called `BioCro:carbon_assimilation_to_biomass` (names are sensitive to capitalization)